### PR TITLE
Lab7

### PR DESCRIPTION
**Pull request by bkmceh**

### CODE REVIEW

**Comment by bkmceh** [comment]
@danmory, @ezio-42

File: monitoring/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    restart: unless-stopped
+    networks:
+      - monitoring.network
+    labels:
+      org.label-schema.group: "monitoring"
+    logging: *loki-logging
+
+  app_python:
+    container_name: app_python
+    image: inforest/app_python:app_python
```
Comment (danmory) [inline monitoring/docker-compose.yml:78]: strange tag

File: monitoring/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    ports:
+      - "5000:5000"
+    restart: unless-stopped
+    logging: *loki-logging
+    networks:
+      - app_python
+
+  app_java:
+    container_name: app_java
+    image: inforest/app_java:app_java
```
Comment (danmory) [inline monitoring/docker-compose.yml:88]: strange tag

**Comment by ezio-42** [comment]
looks good

**Pull request closed without merging**